### PR TITLE
Use bsb.exe instead of node.js wrapper in monorepo

### PR DESCRIPTION
**Pull request by Coobaha**
I've changed bs binaries resolution code a bit, so it resolves to bsb.exe instead of node.js wrapper

### CODE REVIEW

**Comment by JohnPucciarelli** [comment]
@Coobaha were you running into issues with the node.js wrapper?

**Comment by Coobaha** [comment]
Not really, but it is slower to execute it

File: src/com/reason/bs/BsBinaries.java [hunk 1]
```diff
@@ -14,44 +14,29 @@
...
+
+		VirtualFile bscPath = getPathToBinary(virtualFileManager, "bsc", workingDir + LOCAL_BS_PLATFORM);
+		if (bscPath == null) {
+			bscPath = getPathToBinary(virtualFileManager, "bsc", workingDir + LOCAL_NODE_MODULES_BIN);
+		}
+		return bscPath == null ? null : bscPath.getPath();
+	}
 
     @Nullable
     public static String getRefmtPath(@NotNull Project project, @NotNull VirtualFile sourceFile) {
```
Comment (giraud) [inline src/com/reason/bs/BsBinaries.java:42]: Different treatment for refmtPath, ist it wanted ? can it use same resolution mechanism getPathToBinary ?

Comment (Coobaha) [reply src/com/reason/bs/BsBinaries.java:42]: yea, refmt has complete different location and name.. so it is easier to leave it inlined as it was

**Pull request merged**